src/DependencyInjection/Configuration.php

```diff
@@ -148,4 +149,21 @@ private function createMockNode()
 
         return $node;
     }
+
+    private function createToleranceNode()
+    {
+        $treeBuilder = new TreeBuilder();
+        $node = $treeBuilder->root('tolerance');
+
+        $node
+            ->canBeEnabled()
+            ->children()
+                ->scalarNode('retry')->defaultValue(2)->end()
```

Comment:
This means that we can provide a default runner, which would be the retry runner. As you may know, this way we'll have something more flexible, and we could provide other features like throttling.